### ISSUE TITLE
Bundle APK Build Error

### ISSUE THREAD

**Issue by FlexYourBrain** [defold/defold-pbr#3]
When trying to bundle project to .apk I am receiving this build error:

```
/defold-pbr/shaders/pbr.fp
	build/default/defold-pbr/shaders/pbr.fpc:\AppData\Local\Temp\pbr.fpc1788821662676820157.glsl:15: error: 'out' : not allowed in nested scope
:\AppData\Local\Temp\pbr.fpc1788821662676820157.glsl:          "precision mediump int; precision highp float;" 
:\AppData\Local\Temp\pbr.fpc1788821662676820157.glsl:301: error: 'non-opaque uniforms outside a block' : not allowed when using GLSL for Vulkan
:\AppData\Local\Temp\pbr.fpc1788821662676820157.glsl:1397: error: '_DMENGINE_GENERATED_gl_FragColor_0' : undeclared identifier
:\AppData\Local\Temp\pbr.fpc1788821662676820157.glsl:1397: error: 'scalar swizzle' : not supported for this version or the enabled extensions
:\AppData\Local\Temp\pbr.fpc1788821662676820157.glsl:1397: error: 'rgb' : vector swizzle selection out of range
:\AppData\Local\Temp\pbr.fpc1788821662676820157.glsl:1397: error: 'assign' :  cannot convert from ' global highp 3-component vector of float' to ' temp float'
:\AppData\Local\Temp\pbr.fpc1788821662676820157.glsl:1398: error: 'scalar swizzle' : not supported for this version or the enabled extensions
:\AppData\Local\Temp\pbr.fpc1788821662676820157.glsl:1398: error: 'a' : vector swizzle selection out of range
:\AppData\Local\Temp\pbr.fpc1788821662676820157.glsl:1399: error: 'applyDebugMode' : no matching overloaded function found
9 errors generated.
```

Seems to point to > `out vec4 _DMENGINE_GENERATED_gl_FragColor_0;`

[pbr.fpc1788821662676820157.glsl.txt](https://github.com/defold/defold-pbr/files/12381263/pbr.fpc1788821662676820157.glsl.txt)

**Comment by Jhonnyg** [comment]
This should be fixed in defold 1.5 beta, which version are you using?

**Comment by FlexYourBrain** [comment]
Ah nice, I was using 1.4.8. Thanks it does work in 1.5.0
